### ISSUE TITLE
bumper README is misleading

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/bumper#4]
The [README.md](https://github.com/phetsims/bumper/blob/master/README.md) for bumper identifies it as "an educational simulation in HTML5, by PhET Interactive Simulations", and it has an incorrect screenshot.  

I have no idea what this repo is for, but these problems would seem to be related to https://github.com/phetsims/tasks/issues/1018 "Improve high level documentation about phetsims repos".

**Comment by pixelzoom** [comment]
The sim actually looks like this, which appears to be the same as chains:

<img width="1056" alt="screenshot_38" src="https://user-images.githubusercontent.com/3046552/71485683-4d399b80-27e0-11ea-98d7-34917589f191.png">

**Comment by jonathanolson** [comment]
> The README.md for bumper identifies it as "an educational simulation in HTML5, by PhET Interactive Simulations", and it has an incorrect screenshot.

That is likely just the automatically generated readme for anything that looks like a simulation. The repo description on GitHub reads:

> Safely-deployable sim for testing our build/deployment procedures.

Is there a recommended way to better document this? Do we need to change our README generation procedures?

**Comment by pixelzoom** [comment]
I don't know if we need to change any procedures.  Just pointing out that bumper's README.md is a bit misleading/incorrect.   So is chains, examples-sim,...    chains also has the same bogus/bizarre screenshot as bumper, created by @jbphet.

![chains-screenshot](https://user-images.githubusercontent.com/3046552/71627657-ca3e8980-2bc1-11ea-83f3-7a992cce54c5.png)


Assigning to @ariel-phet to decide whether to do anything about this.

**Comment by pixelzoom** [comment]
I asked @jbphet on Slack if there was any significance to the current screenshots for chains and bumper. He replied:
> Nope.  I suspect it was some quickie placeholder thing to get a screenshot in place, which would have been needed to support deployments.
>
> I'd suggest creating an issue to update the screenshots and assigning it to @KatieWoe - she creates most of them these days.

**Comment by pixelzoom** [comment]
@KatieWoe could you please update the screenshots for chains and bumper?

**Comment by pixelzoom** [comment]
@KatieWoe could you please update the screenshots for chains and bumper?  Then please leave this issue assigned to @ariel-phet so he can decided whether the README.md text needs to be changed.

**Comment by KatieWoe** [comment]
How do these look?
![bumper-screenshot](https://user-images.githubusercontent.com/41024075/71698878-6d3a0380-2d7a-11ea-9975-7f3c4218ba4a.png)
![chains-screenshot](https://user-images.githubusercontent.com/41024075/71698879-6dd29a00-2d7a-11ea-9ff0-2acea82c58e7.png)

**Comment by pixelzoom** [comment]
Those look great @KatieWoe, feel free to check them in to GitHub.

**Comment by KatieWoe** [comment]
above commits should do it.

**Comment by ariel-phet** [comment]
Yes, it would be good to change the README from the boiler plate
